### ISSUE TITLE
TypeError: Cannot read property 'aggregate' of undefined

### ISSUE THREAD

**Issue by soulertsai** [dragonman225/notablog#4]
I followed the steps on getting started, yet got errors like this:

![image](https://user-images.githubusercontent.com/7892194/75638545-4fe4c100-5c67-11ea-989d-f0492db641cf.png)

my config.json:
![image](https://user-images.githubusercontent.com/7892194/75638564-6854db80-5c67-11ea-813e-002aff26b778.png)

my notion page:

[link](https://www.notion.so/ea52b5a71a4e4313b826f36ed71228f6?v=a3e3a120ab454266bd794633dbd009d2)

![image](https://user-images.githubusercontent.com/7892194/75638577-7efb3280-5c67-11ea-8575-4eedcbe9fa82.png)

**Comment by soulertsai** [comment]
Another thing, I tried the template link on the getting started page, namely https://www.notion.so/b6fcf809ca5047b89f423948dce013a0?v=03ddc4d6130a47f8b68e74c9d0061de2, it works.

**Comment by dragonman225** [comment]
Notion changes their API recently, so the current version (v0.3.11) do not work.
I'm working on an update to fix it, and for now it is released as a beta package, which you can try with:

```bash
git clone https://github.com/dragonman225/notablog-starter.git -b develop
cd notablog-starter && npm install
```

**Comment by dragonman225** [comment]
This bug is fixed in [v0.4.0](https://github.com/dragonman225/notablog/tree/v0.4.0)
